taskList.txt

```diff
@@ -0,0 +1,4 @@
+T|0|run
+D|1|hop|by sunday
+E|0|laugh|at 2pm
+E|1|hi|at 3pm
```

Comment:
Ideally, you shouldn't commit your data files into the repo, since it will flag unnecessary changes of this file.
It should be ignore by git via its .gitignore.